### PR TITLE
Response to reviewers

### PR DESCRIPTION
**Pull request by mikemahoney218**
This PR will address all comments from the rOpenSci review process; it won't be merged for a bit yet, but I want CI to run against these commits.

### CODE REVIEW

File: vignettes/unifir-user-guide.Rmd [hunk 1]
```diff
@@ -153,6 +165,179 @@ the `set_active_scene()` function) -- to actually _see_ all the changes you've
...
+## A full example
+
+So now that we know the basics of how a unifir script is built, we should walk
+through how to actually build one to make a scene! We'll walk through adding
+terrain, trees, and a character to a scene and then see what it looks like when
+rendered. This example comes from [Will Jones](https://github.com/wjones127), 
+who helped 
+[improve this vignette](https://github.com/mikemahoney218/unifir/issues/7) 
+during 
+[unifir's review at rOpenSci](https://github.com/ropensci/software-review/issues/521).
```
Comment (wjones127) [inline vignettes/unifir-user-guide.Rmd]: I don't think you need to acknowledge me in the vignette itself, but happy to have credit in the git commit message.

```suggestion
rendered.
```

File: vignettes/unifir-user-guide.Rmd [hunk 2]
```diff
@@ -153,6 +165,179 @@ the `set_active_scene()` function) -- to actually _see_ all the changes you've
...
+
+First things first, we'll need a script! Just like before, we'll make our script
+using the `make_script()` function:
+
+```{r eval = FALSE}
+tree_script <- make_script(
+  project = file.path(tempdir(), "unifir", "random_trees")
+  )
```
Comment (wjones127) [inline vignettes/unifir-user-guide.Rmd]:
```suggestion
)
```

File: vignettes/unifir-user-guide.Rmd [hunk 3]
```diff
@@ -153,6 +165,179 @@ the `set_active_scene()` function) -- to actually _see_ all the changes you've
...
+And we'll need to convert that terrain into a format Unity can read, so we'll
+load in the terrainr package as well:
+
+```{r eval = FALSE}
+library(terrainr)
+```
+
+Importing terrain into Unity is a little bit wonky. First off, Unity can't
+process standard raster data, but rather needs a specialized format that
+most libraries don't work with; for that reason, we're going to need to use
```
Comment (wjones127) [inline vignettes/unifir-user-guide.Rmd]: Could you provide the name of the formats? It sounds like Unity needs RAW or possibly TIFF?

**Review by wjones127 — COMMENTED** [review]
Looks good. Had a few minor comments on the user guide, but overall looks a lot better!

File: vignettes/unifir-user-guide.Rmd (hunk 3 above)
Comment (mikemahoney218) [reply vignettes/unifir-user-guide.Rmd]: I added the name of the format -- it is RAW, but unfortunately RAW isn't standardized, so I instead put in the most concise description of the format I'm aware of

**Pull request merged**